### PR TITLE
[ID] Add "Flashlight"

### PR DESCRIPTION
**Pull request by wowcake**
Added the Indonesian article for "Flashlight" game modifier. (https://osu.ppy.sh/wiki/en/Game_modifier/Flashlight)

### CODE REVIEW

File: wiki/Game_modifier/Flashlight/id.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Area yang terlihat akan menjadi lebih kecil pada kombo ke-100x dan lagi pada kombo ke-200x. Jika kombo pemain hancur kapan saja, area yang terlihat akan kembali ke ukuran awalnya. Sebagai tambahan, ketika sedang melakukan sebuah [slider](/wiki/Hit_object/Slider), area yang terlihat akan sebagian digelapkan sampai slider tersebut selesai.
+
+Semua hal-hal tersebut membuat sebuah efek yang kelihatan seperti sebuah senter virtual yang bersinar pada kursor pemain:
+
+![Tampilan permainan FL di osu!](img/FL-comparison-osu.jpg "Perbandingan bermain menggunakan flashlight antara pada kombo ke 13x (atas-tengah), kombo ke 100x (bawah-kiri), dan kombo ke 200x (bawah-kanan)")
+
+Perlu diperhatikan ketika mod ini dipasang dengan mod Hidden juga, area "flashlight" yang terlihat nyaris tidak berguna lagi: dengan pengelihatan yang terbatas, hit object yang memudar ketika kusor tidak difokuskan ke titik posisi hit object.
+
+Mod flashlight sering dikatakan oleh banyak pemain sebagai mod yang paling sulit di osu!, dan skor yang diraih menggunakan mod tersebut biasanya membutuhkan pemain untuk menghafal seluruh bagian beatmap.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:44]:
```suggestion
Mod Flashlight sering dikatakan oleh banyak pemain sebagai mod yang paling sulit di osu!, dan biasanya untuk meraih skor menggunakan mod tersebut, para pemain perlu untuk menghafalkan seluruh bagian beatmap.
```

File: wiki/Game_modifier/Flashlight/id.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### osu!taiko
+
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), posisi area yang terlihat terletak pada area yang digunakan untuk memukul not-notnya. Sama dengan osu!, area yang terlihat semakin mengecil saat kombo yang dimiliki pemain semakin membanyak: memgecil pada kombo ke-100x dan kombo ke-200x dan ketika kombo pemain hancur, ukuran area yang terlihat akan kembali semula.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:48]:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), posisi area yang terlihat terletak pada area yang digunakan untuk memukul not-notnya. Sama dengan osu!, area yang terlihat semakin mengecil saat kombo yang dimiliki pemain semakin banyak: mengecil pada kombo 100x dan kombo 200x dan ketika kombo pemain hancur, ukuran area yang terlihat akan kembali semula.
```

File: wiki/Game_modifier/Flashlight/id.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+![Tampilan permainan FL di taikk](img/FL-taiko.jpg "Tampilan permainan osu!taiko dengan mod Flashlight")
+
+Ketika mod ini dipasang dengan mod Hidden, area "flashlight" yang terlihat menjadi tidak berguna karena secara teknis semua note menjadi "tidak terlihat" karena not-notnya memudar sepenuhnya ketika sampai meraih area yang terlihat. Karena itu, menggunakan kombinasi mod ini membutuhkan pemain untuk menghafal lengkap beatmap tersebut.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:52]:
```suggestion
Ketika mod ini dipasang dengan mod Hidden, area "flashlight" yang terlihat menjadi tidak berguna karena secara teknis semua not menjadi "tidak terlihat" karena not-notnya memudar sepenuhnya ketika sampai meraih area yang terlihat. Karena itu, menggunakan kombinasi mod ini para pemain perlu untuk menghafalkan seluruh bagian beatmap.
```

File: wiki/Game_modifier/Flashlight/id.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+### osu!catch
+
+Pada [osu!catch](/wiki/Game_mode/osu!catch), perilaku mod Flashlight serupa dengan osu!, kecuali daripada area yang terlihat mengikuti kursor, area yang terlihat akan mengikuti si-catcher. Dan karena sifatnya, area yang terlihat menjadi lebih besar dibandingkan dengan osu! ataupun osu!taiko.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:56]:
```suggestion
Pada [osu!catch](/wiki/Game_mode/osu!catch), perilaku mod Flashlight serupa dengan osu!, kecuali pada area yang terlihat akan mengikuti catcher dibandingkan kursor. Dan karena sifatnya, area yang terlihat menjadi lebih besar dibandingkan dengan osu! ataupun osu!taiko.
```

File: wiki/Game_modifier/Flashlight/id.md (hunk 2 above)
Comment (mousesalt) [reply wiki/Game_modifier/Flashlight/id.md:48]: chems

File: wiki/Game_modifier/Flashlight/id.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### osu!mania
+
+Pada [osu!mania](/wiki/Game_mode/osu!mania), area yang terlihat dibatasi pada bar horizontal yang agak tipis di tengah layar sementara hal yang lainnya menjadi tidak terlihat. Dalam artian tersebut, mod Flashlight bisa dianggap sebagai mod Hidden dan mod [Fade In](/wiki/Game_modifier/Fade_In) digabungkan. (Minus, tanpa bagian untuk mengubah ukuran dari area yang terlihat.)
+
+![Tampilan permainan FL di mania](img/FL-mania.jpg "Tampilan permainan osu!mania dengan mod Flashlight")
+
+## Trivia
+
+- Bila pemain berhasil lolos melewati sebuah beatmap dengan nilai S atau SS menggunakan mod Flashlight, pemain akan diberikan nilai variasi perak.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:70]:
```suggestion
- Bila pemain berhasil lolos melewati sebuah beatmap dengan nilai S atau SS menggunakan mod Flashlight, pemain akan diberikan nilai variasi peringkat perak.
```

File: wiki/Game_modifier/Flashlight/id.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- Mod Flashlight pada awalnya menjadi sasaran kontroversi berat mengenai penerapannya pada tahun 2010, hal itu dikarenakan mod ini menjadi mod yang termudah untuk di hack;; dan karena hal tersebut, mod ini terpaksa menjadi unranked sampai sebuah perbaikan dirilis untuk menutupi celah dari mod Flashlight.
```
Comment (mousesalt) [inline wiki/Game_modifier/Flashlight/id.md:71]:
```suggestion
- Mod Flashlight pada awalnya menjadi sasaran kontroversi berat mengenai penerapannya pada tahun 2010, hal itu dikarenakan mod ini menjadi mod yang termudah untuk di hack; dan karena hal tersebut, mod ini terpaksa menjadi unranked sampai sebuah perbaikan dirilis untuk menutupi celah dari mod Flashlight.
```

**Review by mousesalt — COMMENTED** [review]
leaving too many comments, hopefully these is useful

File: wiki/Game_modifier/Flashlight/id.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+- Pengali Skor:
+  - ![][osu!] ![][osu!taiko] ![][osu!catch]: 1.12x
+  - ![][osu!mania]: 1.00x
+- Tombol Shortcut Default: `G`
+- Caption: `Area penglihatan terbatas.`
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Deskripsi
+
+Mod **Flashlight** adalah sebuah [game modifier](/wiki/Game_modifier/Flashlight) yang bertujuan untuk meningkat kesulitan [beatmap](/wiki/Beatmap) dengan memperbatas area yang terlihat pada layar.
```
[the remark this answers is not in the document]
Comment (wowcake) [reply wiki/Game_modifier/Flashlight/id.md:30]: slipped over that one

File: wiki/Game_modifier/Flashlight/id.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+### osu!
+
+Pada [osu!](/wiki/Game_mode/osu!), hanya lingkaran kecil yang diterangi (area yang terlihat) mengelilingi kursor yang akan menampilkan sebagian dari playfield dalam lingkaran tersebut yang akan diperlihatkan. Ukuran dari lingkaran ini akan berubah tergantung dengan kombo yang sedang dimiliki oleh pemain tersebut.
+
+Area yang terlihat akan menjadi lebih kecil pada kombo ke-100x dan lagi pada kombo ke-200x. Jika kombo pemain hancur kapan saja, area yang terlihat akan kembali ke ukuran awalnya. Sebagai tambahan, ketika sedang melakukan sebuah [slider](/wiki/Hit_object/Slider), area yang terlihat akan sebagian digelapkan sampai slider tersebut selesai.
```
[the remark this answers is not in the document]
Comment (wowcake) [reply wiki/Game_modifier/Flashlight/id.md:36]: agree, but not on the ```memainkan slider``` part

**Comment by wowcake** [comment]
turns out the original EN article have a typo on it.
so, probably gonna get that one in aswell.

**Review by mousesalt — APPROVED** [review]

**Pull request merged**